packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -66,5 +66,8 @@
   "force_manifest_editor_show_text":
     "SFDX: Show Package Manifest Editor (Experimental)",
   "show_experimental_webviews_description":
-    "Specifies whether the commands for the experimental webviews are available to the user."
+    "Specifies whether the commands for the experimental webviews are available to the user.",
+    
+  "retrieve_test_code_coverage_text":
```

Comment:
@ruthemmanuelle - Could you help with wording here?

Reply:
Maybe `"Specifies whether code coverage results are calculated and retrieved when you run Apex tests."`?

Reply:
Yes that works. Thanks @ruthemmanuelle. I'll make the changes and merge this in.